app/views/wpcc/your_details/new.html.erb

```diff
@@ -118,24 +124,31 @@
               </div>
             </div>
           </div>
-          <div data-dough-popup-container class="popup-tip__container details__helper">
-            <p data-dough-popup-content class="popup-tip__content"><%= t('wpcc.details.salary.tooltip_html') %></p>
-            <%= render 'wpcc/tooltips/close' %>
-          </div>
+          <%= popup_tip_content options: {
+                                  text: t('wpcc.details.salary.tooltip_html'),
+                                  classname: 'details__helper',
+                                  tooltip_hide: t('wpcc.tooltip_hide')
+          } %>
         </div>
         <div class="details__row--group" data-dough-component="PopupTip">
           <fieldset>
             <legend class="details__calculate-heading"><%= t('wpcc.details.calculate.legend') %></legend>
             <p class="details__calculate-intro">
               <%= t('wpcc.details.calculate.details_html_1') %>
-              <%= render 'wpcc/tooltips/trigger' %>
+              <%= popup_tip_trigger options: {
+                text: t('wpcc.tooltip_show')
+              } %>
               <%= t('wpcc.details.calculate.details_html_2') %>
             </p>
-            <%= render 'wpcc/tooltips/content_qualifying_earnings',
-              formatted_upper_earnings: @your_details_form.formatted_upper_earnings,
-              formatted_lower_earnings: @your_details_form.formatted_lower_earnings,
-              classname: 'details__helper details__helper--block'
-            %>
+            <%= popup_tip_content options: {
+                                    title: t('wpcc.tooltip_qualifying_earnings.title'),
+                                    text: t('wpcc.tooltip_qualifying_earnings.text',
+                                      formatted_upper_earnings: @your_details_form.formatted_upper_earnings,
+                                      formatted_lower_earnings: @your_details_form.formatted_lower_earnings
+                                    ),
+                                    classname: 'details__helper details__helper--block',
+                                    tooltip_hide: t('wpcc.tooltip_hide')
+            } %>
```

Comment:
Same comments as above. The only difference is the classname, is it worth looking to create a helper method and just pass in the classname?